Validation/RecoParticleFlow/python/particleFlowDQM_cff.py

```diff
@@ -90,9 +107,10 @@ def createGenJetPlots(ptbins, etabins):
 
```

Comment:
Please remove thi empty line, for an easier reading

Reply:
Done.